myrestproject/cinema/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import MovieListCreateView, MovieDetailView

```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports) 